packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -82,33 +78,33 @@ class FlatTable<T> extends StatefulWidget {
 
   final ItemCallback<T> onItemSelected;
 
-  final ColumnData<T> sortColumn;
+  final ColumnData<T>? sortColumn;
 
-  final SortDirection sortDirection;
+  final SortDirection? sortDirection;
 
-  final ColumnData<T> secondarySortColumn;
+  final ColumnData<T>? secondarySortColumn;
 
   final Function(
     ColumnData<T> column,
     SortDirection direction, {
-    ColumnData<T> secondarySortColumn,
-  }) onSortChanged;
+    ColumnData<T>? secondarySortColumn,
+  })? onSortChanged;
 
-  final ValueListenable<List<T>> searchMatchesNotifier;
+  final ValueListenable<List<T>>? searchMatchesNotifier;
 
-  final ValueListenable<T> activeSearchMatchNotifier;
+  final ValueListenable<T>? activeSearchMatchNotifier;
 
-  final ValueListenable<T> selectionNotifier;
+  final ValueListenable<T>? selectionNotifier;
 
   int get numSpacers => max(0, columns.length - 1);
 
   @override
   FlatTableState<T> createState() => FlatTableState<T>();
 }
 
-class FlatTableState<T> extends State<FlatTable<T>>
+class FlatTableState<T> extends State<FlatTable<T?>>
     implements SortableTable<T> {
-  List<T> data;
+  List<T>? data;
```

Comment:
this should be late and non-nullable. it is initialized in initState

Reply:
done